### ISSUE TITLE
Web browser continues blocking microphone access due to unsecured connection

### ISSUE THREAD

**Issue by sohansp24** [astrosonic/sanctuary#35]
The Web browser, to protect privacy of the user, keeps blocking microphone access even if permission is allowed. Accessing website using https also doesn't work.

**Comment by sohansp24** [comment]
> The Web browser, to protect privacy of the user, keeps blocking microphone access even if permission is allowed. Accessing website using https also doesn't work.

**Comment by sohansp24** [comment]
> The Web browser, to protect privacy of the user, keeps blocking microphone access even if permission is allowed. Accessing website using https also doesn't work.

**Comment by gridhead** [comment]
> Accessing website using https also doesn't work.

The TLS is yet not implemented on the web app. It is not a requirement as of now but it must be implemented once it gets hosted.

> The Web browser, to protect privacy of the user, keeps blocking microphone access even if permission is allowed.

Can you confirm this with both Chrome and Firefox?

**Comment by sohansp24** [comment]
Yes on both Chrome and Firefox, microphone access is being blocked
